tests/common/plugins/conditional_mark/tests_mark_conditions_skip_traffic_test.yaml

```diff
@@ -61,6 +61,15 @@ decap/test_decap.py:
     conditions:
       - "asic_type in ['vs']"
 
+#######################################
+#####         drop_packets        #####
+#######################################
+drop_packets/test_drop_counters.py:
+  skip_traffic_test:
+    reason: "Skip traffic test for KVM testbed"
+    conditions:
+      - "asic_type in ['vs']"
```

Comment:
Since `test_drop_counters.py` will be skipped on KVM testbed, then why bother making the `skip_traffic_test` change to `test_drop_counters.py`?